mvp-spec/partials-updater/src/files.ts

```diff
@@ -3,12 +3,11 @@ import fs from "fs";
 import path from "path";
 
 const binDir = path.join(RootPath.path, "node_modules", ".bin");
-const assetsDir = path.join(RootPath.path, "..", "assets");
+export const assetsDir = path.join(RootPath.path, "..", "assets");
 const assetPathForDocuments = path.join(".", "assets");
 const partialsDir = path.join(RootPath.path, "..", "partials");
 const documentsDir = path.join(RootPath.path, "..");
-const documentExts = [ ".md" ]
-
+const documentExts = [".md"]
```

Comment:
Do you use a linter with an establisher coding style ? If yes, please add a commit in the README with a pointer to the coding style, else dismiss this line.

Reply:
It's a rather standard coding style, I just format files automatically and apparently that's the only line that didn't match this standard style.
We could add a linter with default config, that would probably be enough.
@kovtun-dmytro-zz maybe? ;-)

Reply:
So, if this is a standard coding style, maybe you can add it in the README so that everybody configures the same.

Reply:
Good idea is to add prettier on the git commit hook. I'll manage it.